lang/en/texts/privacy.html

```diff
@@ -0,0 +1,417 @@
+<p>Privacy Policy</p>
+<p>&nbsp;</p>
+<table>
+<tbody>
+<tr>
+<td colspan="1" rowspan="1">
+<p>In summary:&nbsp;Open Food Facts is a collaborative project whose aim is to promote food transparency by providing an open product database. To make this database available and allow the community to contribute to it, Open Food Facts processes and stores for a limited period, your personal data such as user name, user identifier, passwords, email address and IP for which you have the right of access and information, right of opposition, right of rectification of data, and the right to limitation of processing. If you plan to exercise one of these rights, you can contact us by email: <a href="mailto:privacy@openfoodfacts.org">privacy@openfoodfacts.org</a></p>
+</td>
+</tr>
+</tbody>
+</table>
+<ol start="1">
+<li>
+<h1>Introduction</h1>
+</li>
+</ol>
```

Comment:
It's not allowed to put an h1 inside a ol/li.

Moreover it does not output as you would expect:

![image](https://github.com/openfoodfacts/openfoodfacts-web/assets/144788/e4eb755d-fcfb-4e46-a4a6-917f838ac3de)

Reply:
you even have the summary which is displayed after Introduction, I don't know why !